Packs/AcalvioShadowplex/README.md

```diff
@@ -0,0 +1,9 @@
+Acalvio ShadowPlex is an Autonomous, Distributed Deception Platform designed for enterprise-scale. Deception generates high-fidelity alerts and the integration of ShadowPlex with Cortex XSOAR empowers security teams to respond to incidents with speed and helps to dramatically reduce mean time to respond. The integration also allows the SOC team to leverage XSOAR for managing the deception deployed by ShadowPlex.
```

Comment:
I think it could be better like this
Acalvio ShadowPlex is an Autonomous, Distributed Deception Platform designed for enterprise-scale -> Acalvio ShadowPlex is an Autonomous, Distributed Deception Platform for organizations looking to deploy Deceptions “at Enterprise Scale"

organizations looking to deploy Deceptions “at Enterprise Scale

Reply:
Sure, that sounds good.